.github/workflows/pr-check_redirects_file.yml

```
@@ -0,0 +1,42 @@
+name: Check _redirects.txt file
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  check-redirects:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v3
+        with:
+          node-version-file: ".nvmrc"
+          cache: yarn
+
+      # This is a "required" workflow so path filtering can not be used:
+      # https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/troubleshooting-required-status-checks#handling-skipped-but-required-checks
+      # We have to rely on a custom filtering mechanism to run the checks only if required files are modified.
+      - uses: dorny/paths-filter@v2
+        name: See if any file needs checking
+        id: filter
+        with:
+          filters: |
+            required_files :
+              - "files/en-us/_redirects.txt"
```

```suggestion
              - ".nvmrc"
              - "files/**"
```
Handle the [edge case](https://github.com/mdn/content/pull/28277#issuecomment-1658653879)